### ISSUE TITLE
www.amazon.* URL Tracking filter part II

### ISSUE THREAD

**Issue by Morku** [AdguardTeam/AdguardFilters#105213]
### Prerequisites

- [X] This site DOES NOT contains sexually explicit material, otherwise use [NSFW-specific form](./new?template=bug_report_NSFW.yml);
- [X] Filters were updated before reproducing an issue;
- [X] AdGuard product version is up-to-date;
- [X] Browser version is up-to-date;
- [X] If the site or app is broken, disabling AdGuard protection resolves an issue.

### What product do you use?

AdGuard for Windows

### AdGuard version

7.8

### What type of problem have you encountered?

Missed analytics or tracker

### Which browser(s) do you use?

Chrome

### Which device do you use?

Desktop

### Where is the problem encountered?

https://www.amazon.de/KZ-Ear-Headset-Einheiten-HiFi-Bass-Ohrh%C3%B6rer-Cancelling-Ohrh%C3%B6rer-Schwarz/dp/B07QYKSKDN/ref=asc_df_B07QYKSKDN/?tag=googshopde-21&linkCode=df0&hvadid=309826494914&hvpos=&hvnetw=g&hvrand=11316320153620431089&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9068546&hvtargid=pla-718670810796&psc=1&th=1&psc=1&tag=&ref=&adgrpid=63149895578&hvpone=&hvptwo=&hvadid=309826494914&hvpos=&hvnetw=g&hvrand=11316320153620431089&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9068546&hvtargid=pla-718670810796

### What filters do you have enabled?

AdGuard Base filter, AdGuard Mobile Ads filter, AdGuard German filter, AdGuard Tracking Protection filter, AdGuard URL Tracking filter, AdGuard Social Media filter, AdGuard Annoyances filter, AdGuard DNS filter

### What Stealth Mode options do you have enabled?

Hide your search queries, Send Do-Not-Track header

### Add your comment and screenshots

1. 
When I reported Amazon Tracking parameter here https://github.com/AdguardTeam/AdguardFilters/issues/85274 I also reported some more I did encounter by getting an Amazon URL, without knowing how to reproduce the link here: https://github.com/AdguardTeam/AdguardFilters/issues/85274#issuecomment-858745669

I know how to reproduce now:
- disable Adblocking
- go to Google
- search fo a product sold on Amazon e.g. "kz pro"
- click on the Amazon result of "Ad/Anzeige"
- you will get a result like the link here.

You may think "but when you use Adblocker, you don't get the Ad result and the normal search result don't contain tracking parameter". Thats right. But when friends and family may share an URL with you, they don't debloat it and I have to copy/paste the useful part of URL out of it.
Can you check again the porpuse of the URL parameter? Maybe they have any use...? Some appear empty in provided example 🤷‍♂️

In URL you see `tag` `linkCode` `hvadid` `hvpos` `hvnetw` `hvrand` `hvpone` `hvptwo` `hvqmt` `hvdev` `hvdvcmdl` `hvlocint` `hvlocphy` `hvtargid` `psc` `th` `adgrpid` `hvqmt`

2. Video
<details><summary>Video 1:</summary>

https://user-images.githubusercontent.com/6276915/147831741-0e6808e8-4600-44ac-8533-717228ec9ba5.mp4

</details>

### Privacy

- [X] I agree to follow this condition

**Comment by Yuki2718** [comment]
I've personally been removing `linkCode` used on many affiliate links, but no idea about other parameters here.

**Comment by Morku** [comment]
@Alex-302 
Why closing without feedback?

@BlazDT 
You checked the Amazon URL Tracking last time. Is this no tracking and have it any use?

**Comment by Alex-302** [comment]
Do you know the purpose of every of these parameters?

**Comment by BlazDT** [comment]
I do not know those parameter yet. In general some are required to display action you bought an article already, some are required in chat.

**Comment by Morku** [comment]
@Alex-302 
No, thats why I asked "Can you check again the porpuse of the URL parameter?". So a short "unknown purpose, could break something" would be nice before closing :)
